packages/bridge-ui/src/vault/tokenVaults.ts

```diff
@@ -1,10 +1,12 @@
+import type { Address } from '@wagmi/core';
+
```

Comment:
```suggestion
```

Reply:
Nope. I'm using this plugging to sort and organize the imports, https://github.com/lydell/eslint-plugin-simple-import-sort